src/main/java/entities/TaskFactory.java

```diff
@@ -0,0 +1,13 @@
+package entities;
+
+public class TaskFactory {
+
+    public Task create(String name, User user) {
+        return new Task(name, user);
+    }
+    
+    public Task create(String name, User user, String description) {
```

Comment:
I have a question: is the factory really necessary if all it does is call the corresponding method on the Task entity? There is only one type of Task, so would it not be more convenient to directly call the initialization method in Task?

Reply:
I think it's good design.